.dockerignore

```diff
@@ -0,0 +1,17 @@
+**/__pycache__/
```

Comment:
These temporary files were bloating the size of the context that's getting fed into image builds.